cmd/restic/cmd_stats.go

```diff
@@ -282,9 +308,15 @@ func verifyStatsInput(gopts GlobalOptions, args []string) error {
 // to collect information about it, as well as state needed
 // for a successful and efficient walk.
 type statsContainer struct {
-	TotalSize      uint64 `json:"total_size"`
-	TotalFileCount uint64 `json:"total_file_count"`
-	TotalBlobCount uint64 `json:"total_blob_count,omitempty"`
+	TotalSize                            uint64  `json:"total_size"`
+	TotalUncompressedSize                uint64  `json:"total_uncompressed_size,omitempty"`
+	TotalCompressedBlobsSize             uint64  `json:"total_compressed_blobs_size,omitempty"`
+	TotalCompressedBlobsUncompressedSize uint64  `json:"total_compressed_blobs_uncompressed_size,omitempty"`
```

Comment:
The JSON output shouldn't contain more fields than the text output, please just exclude `TotalCompressedBlobsSize` and `TotalCompressedBlobsUncompressedSize` from the json output.

Reply:
Done.